login-workflow/example/src/screens/Login.tsx

```diff
@@ -3,19 +3,25 @@ import { LoginScreen, useAuthContext } from '@brightlayer-ui/react-auth-workflow
 import EatonLogo from '../assets/images/eaton_stacked_logo.png';
 import { useApp } from '../contexts/AppContextProvider';
 import { DebugComponent } from '../components/DebugComponent';
+import { useLocation } from 'react-router-dom';
 
 export const Login = (): JSX.Element => {
     const { setIsAuthenticated } = useApp();
     const auth = useAuthContext();
-
+    const location = useLocation();
     return (
         <>
             <LoginScreen
                 onLogin={(username, password): void => {
                     // eslint-disable-next-line no-console
                     console.log('onLogin', username, password);
                     setIsAuthenticated(true);
-                    auth.navigate('homepage');
+
+                    if (location.state?.from.pathname) {
```

Comment:
For navigating to original page after login

Reply:
When there is authenticated route location.state?.from.pathname will be undefined. If any route redirect from guard component then only it is having value. 

To fix routes which are not defined I am redirecting to <PageNotFound /> component.